Jenkinsfile

```diff
@@ -294,6 +294,29 @@ pipeline {
       }
     }
 
+    stage('L2: Parallel Stage QuartzNet/JasperNet inference') {
+      when {
+        anyOf{
+          branch 'master'
+          changeRequest()
+        }
+      }
+      failFast true
+      parallel {
+        stage('QuartzNet inference') {
+          steps {
+            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=0 python speech2text_infer.py --asr_model=QuartzNet15x5-En --dataset=/home/TestData/librispeech/librivox-dev-other.json --wer_target=0.1060'
+          }
+        }
+        stage('JasperNet inference') {
+          steps {
+            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=0 python speech2text_infer.py --asr_model=JasperNet10x5-En --dataset=/home/TestData/librispeech/librivox-dev-other.json --wer_target=0.1041'
```

Comment:
CUDA_VISIBLE_DEVICES=1?

Or is it for a reason you want to use the same GPU?

Reply:
yes, bug - should be =1. fixed